src/components/actions/scheduler.ts

```diff
@@ -53,15 +61,46 @@ export const triggerScheduledActions = async () => {
   DBConnect.triggerScheduledActions()
 }
 
-export const cleanUpPreviewFiles = async () => {
+const checkFile = async (filePath: string) => {
+  if (await DBConnect.checkIfInFileTable(filePath)) {
+    const subPath = filePath.slice(basePath.length)
+    const file: any = { filePath: subPath }
```

Comment:
Try not to use `any`. This is a good example of why not. There was a weird error occurring when running the script which I couldn't easily figure out. But as soon as I changed this `any` to a specific type (`{filePath: string}`) it became immediately obvious what the problem was -- this is not the correct input type for the `deleteFile` function (it needed thumbnailPath and a couple of other things). I was able to modify the "deleteFile" function to handle different input, but it was so much easier to figure out what the problem was once Typescript alerted me to it.